app/src/main/java/com/example/she/addrelative.java

```diff
@@ -65,7 +65,15 @@ public void onClick(View v) {
 
             }
         });
+    }
 
-
+    public boolean checkPhoneNumber(){
+        String number = edt_num.getText().toString().trim();
+        if(number.length() < 10){
```

Comment:
What if user enters more than 10 digit? Change the condition to == 10 , so that only numbers containing 10 digits are saved.

Reply:
Can I update this pull request for the above but then the commits will become two. Will this be ok ?

Reply:
> Can I update this pull request for the above but then the commits will become two. Will this be ok ?

Okay.

Reply:
Updated the condition to make sure the digits of phone number is equal to 10.